packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -55,5 +55,6 @@
   "isv_bootstrap_command_text":
     "SFDX: Create and Set Up Project for ISV Debugging",
   "isv_debugger_enabled_description":
-    "Set to true to enable a feature preview of the ISV Debugger."
+    "Set to true to enable a feature preview of the ISV Debugger.",
+  "force_apex_log_fetch_text": "SFDX: Fetch Apex Debug Logs..."
 }
```

Comment:
@ruthemmanuelle please take a look at this message

Reply:
Looks fine to me. Thanks for pointing it out.

Reply:
The other messages look fine too.